env.py

```diff
@@ -55,7 +55,7 @@ def get_action_space(self):
         raise NotImplementedError
 
     @abstractmethod
-    def get_mask(self):
+    def get_mask_invalid_actions(self):
```

Comment:
Why two methods with the same name, one with `@abstractmethod`?

Reply:
Because one is in the EnvBase class, and the other is the aptamer specific implementation